sdk/identity/Azure.Identity/images/mermaidjs/DefaultAzureCredentialAuthFlow.md

```diff
@@ -18,9 +17,9 @@ flowchart LR;
     end;
 
     %% Define styles for credential type boxes
-    classDef deployed fill:#71AD4C, stroke:#71AD4C;
-    classDef developer fill:#EB7C39, stroke:#EB7C39;
-    classDef interactive fill:#A6A6A6, stroke:#A6A6A6;
+    classDef deployed fill:#95C37E, stroke:#71AD4C;
```

Comment:
The preview seems to show that the contrast is less than it was before.

Reply:
It's the same color scheme I used for the other languages. For example, see Go at https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/azidentity#defaultazurecredential.

Reply:
Ah ok- I think the confusion is that the new color is lighter, but in contrast to a dark foreground text, it is higher contrast.